src/overrides/gemini-scrollbar.less

```diff
@@ -1,4 +1,4 @@
-@import (inline) "../../node_modules/react-gemini-scrollbar/node_modules/gemini-scrollbar/gemini-scrollbar.css";
+@import (inline) "../../../../node_modules/react-gemini-scrollbar/node_modules/gemini-scrollbar/gemini-scrollbar.css";
```

Comment:
Fixed, now people using reactjs components will be required to add it themselves.